src/Features/Lsif/Generator/Writing/ILsifJsonWriter.cs

```diff
@@ -2,12 +2,14 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Collections.Generic;
 using Microsoft.CodeAnalysis.LanguageServerIndexFormat.Generator.Graph;
 
 namespace Microsoft.CodeAnalysis.LanguageServerIndexFormat.Generator.Writing
 {
     internal interface ILsifJsonWriter
     {
         void Write(Element element);
```

Comment:
❔ Do we still need this one?

Reply:
yup.  several callers call directly into that one.  I could have them wrap into a single-element list, but it seemed wasteful :)